prompt-service/src/unstract/prompt_service/helper.py

```diff
@@ -80,3 +81,58 @@ def plugin_loader(app: Flask) -> dict[str, dict[str, Any]]:
         app.logger.info("No plugins found.")
 
     return plugins
+
+
+def query_usage_metadata(db, run_id, token):
```

Comment:
Are we directly query one DB from prompt service? I was wondering we usually do it via platform service?

Reply:
It is done here because I need to attach these data along with prompt service. I noticed that DB connection is already there in prompt-service and used it. 

Also not all query goes through platform service. Check this for reference.
https://github.com/Zipstack/unstract/blob/49b9873a7a3feca769919fe2ce8846c743c187fd/prompt-service/src/unstract/prompt_service/authentication_middleware.py#L58